README.md

```diff
@@ -32,14 +32,17 @@ In order to run strings you need to have the following software installed:
   * Puppet 3.7 or newer
   * The yard rubygem
 
-Install the yard rubygem with the gem command:
+Install the yard rubygem to Puppet's Gem set.
 
+On *nix systems, run:
 ```
-$ gem install yard
```

Comment:
And as I mentioned in hipchat, maybe have a section for installing the gem on 3.7+ and a separate snippet for 4.x.